### PR TITLE
chore: Remove monaco deps from script

### PR DESCRIPTION
**Pull request by richburdon**
- [x] Remove monaco.
- [x] Use hooks instead of component for script editor. 
- [ ] Remove useTextModel.

### CODE REVIEW

File: packages/apps/plugins/plugin-script/src/ScriptPlugin.tsx [hunk 1]
```diff
@@ -100,30 +101,30 @@ export const ScriptPlugin = ({ containerUrl }: ScriptPluginProps): PluginDefinit
             case 'main':
               return isObject(data.active, ScriptType.schema, ScriptType.filter()) ? (
                 <Main.Content classNames={[baseSurface, fixedInsetFlexLayout, topbarBlockPaddingStart]}>
-                  <ScriptBlock
-                    id={(data.active as any).id}
-                    source={(data.active as ScriptType).source}
-                    classes={{ toolbar: 'px-2' }}
+                  <ScriptBlockWrapper
+                    //
```
Comment (dmaretskyi) [inline packages/apps/plugins/plugin-script/src/ScriptPlugin.tsx:105]: //

File: packages/apps/plugins/plugin-script/src/ScriptPlugin.tsx [hunk 2]
```diff
@@ -100,30 +101,30 @@ export const ScriptPlugin = ({ containerUrl }: ScriptPluginProps): PluginDefinit
...
                 </Main.Content>
               ) : null;
             case 'slide':
               return isObject(data.slide, ScriptType.schema, ScriptType.filter()) ? (
-                <ScriptBlock
-                  id={(data.slide as any).id}
-                  source={(data.slide as ScriptType).source}
-                  classes={{ root: 'p-24' }}
+                <ScriptBlockWrapper
+                  //
```
Comment (dmaretskyi) [inline packages/apps/plugins/plugin-script/src/ScriptPlugin.tsx:115]: //

File: packages/apps/plugins/plugin-script/src/ScriptPlugin.tsx [hunk 3]
```diff
@@ -100,30 +101,30 @@ export const ScriptPlugin = ({ containerUrl }: ScriptPluginProps): PluginDefinit
...
-                  containerUrl={containerUrl}
                 />
               ) : null;
             case 'section':
               return isObject(data.object, ScriptType.schema, ScriptType.filter()) ? (
-                <ScriptBlock
-                  id={(data.object as any).id}
-                  source={(data.object as ScriptType).source}
+                <ScriptBlockWrapper
+                  //
```
Comment (dmaretskyi) [inline packages/apps/plugins/plugin-script/src/ScriptPlugin.tsx:126]: //

File: packages/apps/plugins/plugin-script/src/components/ScriptEditor/ScriptEditor.stories.tsx [hunk 4]
```diff
@@ -2,32 +2,61 @@
...
 import { TextObject } from '@dxos/client/echo';
+import { createDocAccessor, type DocAccessor } from '@dxos/echo-schema';
 import { TextKind } from '@dxos/protocols/proto/dxos/echo/model/text';
 
 import { ScriptEditor } from './ScriptEditor';
 
-// prettier-ignore
-const code = [
+const example1 = [
+  //
```
Comment (dmaretskyi) [inline packages/apps/plugins/plugin-script/src/components/ScriptEditor/ScriptEditor.stories.tsx:27]: //

File: packages/apps/plugins/plugin-script/src/components/ScriptEditor/ScriptEditor.stories.tsx [hunk 5]
```diff
@@ -2,32 +2,61 @@
...
   'export default function() {',
   '  const value = 100',
   '  return <div>{value}</div>;',
   '}',
 ].join('\n');
 
+const example2 = [
+  //
```
Comment (dmaretskyi) [inline packages/apps/plugins/plugin-script/src/components/ScriptEditor/ScriptEditor.stories.tsx:35]: //

File: packages/core/echo/echo-schema/src/automerge/automerge-object-core.ts [hunk 6]
```diff
@@ -378,22 +379,36 @@ export class AutomergeObjectCore {
   }
 }
 
-export type DocAccessor<T = any> = {
+const isDocument = Symbol.for('isDocument');
+
+// TODO(burdon): Rename ValueAccessor.
```
Comment (dmaretskyi) [inline packages/core/echo/echo-schema/src/automerge/automerge-object-core.ts:384]: I dont really like this name. Maybe CrdtAccessor

File: packages/ui/react-ui-editor/src/hooks/useTextEditor.ts [hunk 7]
```diff
@@ -88,7 +84,9 @@ export const useTextEditor = ({
     return () => {
       view?.destroy();
     };
-  }, [doc, extensions]);
+  }, deps);
+
+  useDebugReactDeps([doc, ...(deps ?? [])]);
```
Comment (dmaretskyi) [inline packages/ui/react-ui-editor/src/hooks/useTextEditor.ts:89]: remove

File: packages/ui/react-ui-editor/src/hooks/useTextEditor.ts [hunk 8]
```diff
@@ -155,3 +153,18 @@ export const createThemeExtensions = ({ theme, themeMode, slots }: ThemeExtensio
     EditorView.contentAttributes.of({ class: slots?.content?.className ?? '' }),
   ].filter(isNotFalsy);
 };
+
+const useDebugReactDeps = (deps: DependencyList = []) => {
```
Comment (dmaretskyi) [inline packages/ui/react-ui-editor/src/hooks/useTextEditor.ts:157]: extract to a testing file?

File: packages/apps/plugins/plugin-script/src/ScriptPlugin.tsx (hunk 1 above)
Comment (richburdon) [reply packages/apps/plugins/plugin-script/src/ScriptPlugin.tsx:105]: this is stop prettier from making the code less pretty.

**Pull request merged**